src/views/Records/Records.vue

```diff
@@ -142,85 +124,88 @@ export default {
     }
   },
   mounted: function () {
+    window.addEventListener("scroll", this.onScroll);
```

Comment:
Im really starting to think we shoumd rely on vue-scrollTo which is much easier to use and integrate into tests. It's what I've used in the new edit panels.

Reply:
Okay I will check to see how I can use it in this case

Reply:
I checked that. Due to we are triggering window it is better to stick to current classic way of using window.addeventListener one because anyway I am checking window property such as window.top.scrollY in onScroll function which is already working properly. Besides, vueScrollTo needs a container which by default is body not window. window is a higher object level in DOM I believe.